pybossa/cache/projects.py

```diff
@@ -161,6 +161,18 @@ def n_tasks(project_id):
     return n_tasks
 
 
+@memoize(timeout=timeouts.get('APP_TIMEOUT'), cache_group_keys=[[0]])
+def n_tasks_not_gold(project_id):
+    """Return number of tasks of a project."""
+    sql = text('''SELECT COUNT(task.id) AS n_tasks FROM task
+                  WHERE task.project_id=:project_id AND calibration = 0;''')
```

Comment:
task is gold when calibration is 1. though this condition would work, it can be 
```python
calibration != 1
```
this will cover any bad value other than 1 to be not treated as gold task.